radiant/mainframe.cpp

```diff
@@ -830,9 +836,9 @@ static gint mainframe_keypress( GtkWidget* widget, GdkEventKey* event, gpointer
 static gint mainframe_keyrelease( GtkWidget* widget, GdkEventKey* event, gpointer data ){
 	unsigned int code = gdk_keyval_to_upper( event->keyval );
 
-	if ( gtk_accelerator_valid( event->keyval, (GdkModifierType)0 ) ) {
-		return TRUE;
-	}
+//	if ( gtk_accelerator_valid( event->keyval, (GdkModifierType)0 ) ) {
+//		return TRUE;
+//	}
```

Comment:
I _think_ this was just a "performance optimization"? It needs to be commented out in order to send keyup events to non-special keys, which is required for the new smooth camera movement code. I did not notice any negative side effects by commenting this out.